### PR TITLE
Node-Red Add TZ variable

### PR DESCRIPTION
**Pull request by Adrian-Grimm**
TimeZone env added to Node-Red - solves #2721

### CODE REVIEW

File: library/ix-dev/community/node-red/questions.yaml [hunk 1]
```diff
@@ -21,6 +21,15 @@ portals:
     path: "$kubernetes-resource_configmap_portal_path"
 
 questions:
+  - variable: TZ
+    group: Home Assistant Configuration
```
Comment (stavros-k) [inline library/ix-dev/community/node-red/questions.yaml:25]: Please fix group

**Comment by Adrian-Grimm** [comment]
@stavros-k: fixed.

File: library/ix-dev/community/node-red/questions.yaml [hunk 2]
```diff
@@ -21,6 +21,15 @@ portals:
     path: "$kubernetes-resource_configmap_portal_path"
 
 questions:
+  - variable: TZ
+    group: Node-RED Configuration
+    label: Timezone
+    schema:
+      type: string
+      default: America/Los_Angeles
```
Comment (stavros-k) [inline library/ix-dev/community/node-red/questions.yaml:29]: Let's be consistent with the rest and default to Etc/UTC

Comment (Adrian-Grimm) [reply library/ix-dev/community/node-red/questions.yaml]: OK - Fixed

**Pull request merged**